tests/adapters/switches/dell10g_test.py

```diff
@@ -32,6 +32,7 @@
 from netman.core.objects.switch_transactional import FlowControlSwitch
 
 
+@mock.patch("netman.adapters.switches.dell.warnings.warn", mock.Mock())
```

Comment:
Could we make deprecations warning inherits from a common ancestor and use a @IgnoreDeprecationWarnings decorator or something like that to ignore deprecations warning ?